src/modules/avformat/link_avdeinterlace.c

```diff
@@ -62,6 +62,13 @@ static mlt_image_format validate_format(mlt_image_format format)
     case mlt_image_yuv420p10:
     case mlt_image_yuv444p10:
         ret_format = format;
+        break;
+    case mlt_image_none:
+    case mlt_image_movit:
+    case mlt_image_opengl_texture:
+    case mlt_image_invalid:
+        ret_format = mlt_image_yuv422;
```

Comment:
Why `mlt_image_yuv422` here and not `mlt_image_rgba` per my recent fix in b365ce645 ?

Reply:
This change follows the existing behavior. Previously, it would have returned the value set on line 55 above. My goal was resolve the warnings and not speculatively optimize. When I did the initial implementation, FFMpeg always seemed to want to convert to YUV for the deinterlacers. So I thought that might avoid some conversions. Maybe for Movit/OpenGL we should have FFMpeg do the YUV to RGBA conversion. I can make that change if you think that will be best.

Reply:
No, don’t change at this time. Oversight on my review.